json2timeseriesdb

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env ruby
 
+# Keep these two scripts in sync.
+# puppetlabs-puppet-metrics-viewer/json2timeseriesdb
```

Comment:
This viewer is just a wrapper around existing tools... why not just have this script install the collector and use the file directly from it?  

Or alternatively curl the file down from github instead of duplicating?

Reply:
The long-term goal is to merge Puppet Metrics Viewer and Puppet Metrics Dashboard, eliminating the duplicate scripts (along with a litany of other duplication). The short-term goal is to merge the changes to the scripts, and document the duplication for anyone who needs to update the script. Charlie suggests that since Puppet Metrics Viewer uses Docker, and there is work to move the tests in Puppet Metrics Dashboard to Limtus, we may be able to merge the two, and users (like Support) could reuse the acceptance test to spin up Puppet Metrics Dashboard in Docker via Litmus, allowing us to merge Puppet Metrics Viewer and Puppet Metrics Dashboard.

Reply:
We do already curl down the dashboards from the `puppet_metrics_dashboard` so that would be a logical extension to use with the `json2timeseriesdb` file.

Reply:
Magic!

Reply:
So if I understand correctly ...

* Delete the duplicate `json2graphite.rb` script from this repository.
* Download the `json2timeseriesdb` script from the `puppet_metrics_collector` repository in `view-in-grafana.sh` like this:

```
download_dashboards() {
  download_file https://raw.githubusercontent.com/puppetlabs/puppetlabs-puppet_metrics_collector/master/files/json2timeseriesdb
  chmod +x json2timeseriesdb

  mkdir -p ./grafana/imports
  cd ./grafana/imports || exit
  download_file https://raw.githubusercontent.com/puppetlabs/puppet_metrics_dashboard/master/files/PuppetDB_Performance.json
  download_file https://raw.githubusercontent.com/puppetlabs/puppet_metrics_dashboard/master/files/PuppetDB_Workload.json
  download_file https://raw.githubusercontent.com/puppetlabs/puppet_metrics_dashboard/master/files/Puppetserver_Performance.json
  download_file https://raw.githubusercontent.com/puppetlabs/puppet_metrics_dashboard/master/files/Archive_File_Sync.json
  cd - || exit > /dev/null
}
```

* Rename all references to `json2graphite.rb` to `json2timeseriesdb` in `README.md` and `view-in-grafana.sh`

Yes?

Reply:
See: https://github.com/puppetlabs/puppet-metrics-viewer/pull/38